integration/setup/startup.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package setup provides integration tests setup helpers.
+package setup
+
+import (
+	"sync/atomic"
+)
+
+// ports are available port of Tigris.
+var ports = []uint16{8081, 8082, 8083, 8085, 8086}
```

Comment:
I like this idea, but I'm a bit skeptical about these ports being defined here as the exact number of exact ports and not being configurable.
Is there a plan to make those things configurable (like `tigris-url` itself) - maybe in separate PRs?

Reply:
I've looked at it briefly and decided I would not do it for this, currently there were several places where ports are defined, env tool, docker-compose.yml, and within integrations tests. So it's nice to have configurable but also it touches more than the integration test parameter.  For this PR, I'm gonna do quick win. :see_no_evil:

Reply:
Oh, `envtool` is a nice catch as we wait for `8081` only, and not for the other ports...  
I'm ok to merge it as is to have a quick win, and maybe we can reconsider this and make it configurable after #1857 (so, it'll be easier to deal with configuration).

Reply:
I think we should start paying a bit more attention to our code comments and not leave explanations on GitHub only, where they are unlikely to be found.

In this case, cross-linking those ports to the Docker Compose configuration would significantly improve [understanding](https://www.osnews.com/story/19266/wtfsm/).

Reply:
Yeah agree. I've added comments. I also foresee the number of instances are just a guess and need adjusting hence a new issue https://github.com/FerretDB/FerretDB/issues/1887

Reply:
See bd05d83c9d5df8e36f41de8aff7a95bb60ed215b